### PR TITLE
render data and render menu

### PR DESCRIPTION
**Pull request by aika-l**
task#2 and task#3 (Render Data and Render Menu)

### CODE REVIEW

**Comment by aika-l** [comment]
adding this branch to group4 branch

**Comment by maratgaip** [comment]
> task #1 and task#2

i don't see task 1. Please refactor your code.

File: script.js [hunk 1]
```diff
@@ -1 +1,27 @@
-// your code
+let center=document.querySelector(".section-center")
```
Comment (maratgaip) [inline script.js:1]: 1. const would be better.
2. Please use space before and after any math operator.
3. Use better name

const  center = document.querySelector(".section-center")

File: script.js [hunk 2]
```diff
@@ -1 +1,27 @@
...
+
+window.addEventListener("DOMContentLoaded",()=>{
+    fetch ('https://gist.githubusercontent.com/maratgaip/44060c688fcf5f2b7b3985a6d15fdb1d/raw/e93c3dce0826d08c8c6e779cb5e6d9512c8fdced/restaurant-menu.json')
+
+    .then(response=>response.json())
+    .then(data=>renderData(data))
```
Comment (maratgaip) [inline script.js:7]: .then(renderData) would enough.

Add comments to your code.

File: script.js [hunk 3]
```diff
@@ -1 +1,27 @@
...
+})
+
+function renderData(menuItem){
+let menuData=menuItem.map(item=>{
+    return ` <article class="menu-item">
```
Comment (maratgaip) [inline script.js:12]: use destructure.

**Review by maratgaip — CHANGES_REQUESTED** [review]

File: script.js (hunk 1 above)
Comment (aika-l) [reply script.js:1]: 1. updated let to const and gave more meaning full name
2. added spaces per recommendation

File: script.js (hunk 2 above)
Comment (aika-l) [reply script.js:7]: comments added

**Pull request merged**